### PR TITLE
(#194) Update nexus-repository package upgrade instructions

### PR DESCRIPTION
**Pull request by steviecoaster**
The new Nexus package incorporates the ability to seamlessly update a Nexus configuration that
utilizes SSL. As such, updated the documentation to provide an example one-line command that is
now possible to use to upgrade this package.

### CODE REVIEW

**Comment by steviecoaster** [comment]
NOTE TO REVIEWERS: 

Don't switch this off of Draft until the new `nexus-repository` package changes are merged on the package source repository. That way these docs hit the site at the same time as the package.

File: input/en-us/quick-deployment/setup/upgrade-nexus-qde.md [hunk 1]
```diff
@@ -11,58 +11,28 @@ This document outlines the process for upgrading Nexus running inside our Quick
...
 # Instructions
 
 1. Internalize the nexus-repository package and push to your internal repo
-2. Stop the nexus service
-3. Backup any SSL configuration if it exists
-4. Remove the C:\ProgramData\nexus folder recursively
-5. choco upgrade the nexus-repository package
-6. Restore the SSL configuration if it exists
-7. Restart the nexus service
+2. choco upgrade the nexus-repository package
```
Comment (ryanrichter94) [inline input/en-us/quick-deployment/setup/upgrade-nexus-qde.md:23]:
```suggestion
2. choco upgrade the nexus-repository package (Upgrade command example given below)
```

Comment (ryanrichter94) [reply input/en-us/quick-deployment/setup/upgrade-nexus-qde.md:23]: Added this for fear of people not reading further and just running `choco upgrade nexus-repository` with no package parameters.

Comment (steviecoaster) [reply input/en-us/quick-deployment/setup/upgrade-nexus-qde.md]: Used slightly different wording, but addressed this. Force pushed change

**Review by ryanrichter94 — APPROVED** [review]
LGTM 🚀

**Pull request merged**